src/credentials-manager/index.ts

```diff
@@ -65,16 +53,16 @@ class CredentialsManager {
   /**
    * Gets the credentials that has already been saved
    *
-   * @param {String} scope optional - the scope to request for the access token. If null is passed, the previous scope will be kept.
-   * @param {String} minTtl optional - the minimum time in seconds that the access token should last before expiration.
-   * @param {Object} parameters optional - additional parameters to send in the request to refresh expired credentials.
-   * @param {Object} forceRefresh optional - to force refresh the credentials. It will work only if the refresh token already exists.
-   * @returns {Promise}
+   * @param scope The scope to request for the access token. If null is passed, the previous scope will be kept.
+   * @param minTtl The minimum time in seconds that the access token should last before expiration.
+   * @param parameters Additional parameters to send in the request to refresh expired credentials.
+   * @param forceRefresh  Whether to force refresh the credentials. It will work only if the refresh token already exists.
+   * @returns A populated instance of {@link Credentials}.
    */
   async getCredentials(
     scope?: string,
     minTtl: number = 0,
-    parameters: object = {},
+    parameters: Record<string, unknown> = {},
```

Comment:
I've changed a few of these - `Record<string, unknown>` feels much better than just `object`.